Add timing and logging for Elasticsearch calls

martint opened the pull request.

martint committed:
Add timing and logging for Elasticsearch calls

electrum commented on presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ConnectorObjectNameGeneratorModule.java:
```diff
@@ -0,0 +1,117 @@
+import static io.airlift.configuration.ConfigBinder.configBinder;
+import static java.util.Objects.requireNonNull;
+
+// Note: There are multiple copies of this class in the codebase.  If you change one you, should change them all.
```
Maybe move to plugin toolkit

martint committed:
Move ConnectorObjectNameGenerator to plugin toolkit

We already have too many copies of the class and we'll be adding
more usages soon.

martint commented:
@electrum, updated
